internal/cue/fixtures/valid.yaml

```diff
@@ -11,15 +11,26 @@ flags:
     name: flipt
   rules:
   - segment: internal-users
-    rank: 1
```

Comment:
just FYI, `fixtures` has been renamed to `testdata` on main so that it works with fuzz testing